recipes/RigolWFM/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "RigolWFM" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/RigolWFM-{{ version }}.tar.gz
+  sha256: 85ae9dafd7e6d4b826fa73d7180e61610d563392869a2abbae24d827cb84895b
+
+build:
+  entry_points:
+    - wfmconvert = RigolWFM.wfmconvert:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools >=44
+  run:
+    - kaitaistruct
+    - matplotlib-base
+    - numpy
+    - python >=3.6
+    - requests
+
+test:
+  imports:
+    - RigolWFM
+  commands:
+    - pip check
+    - wfmconvert --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/scottprahl/RigolWFM.git
+  summary: |
+    A comprehensive resource for interpreting waveform ``.wmf`` files created
+    by any Rigol oscilloscope.s
```

Comment:
```suggestion
    by any Rigol oscilloscopes.
```

Reply:
"any" is singular.  "oscilloscope" is correct.  Thanks for the review.

Reply:
You mean to keep that `.s`? Usually we expect the summary to be the same as GH:

![Screenshot from 2023-08-18 17-01-55](https://github.com/conda-forge/staged-recipes/assets/950575/224d2dd5-d1c7-40a6-9ea9-a73dfca30927)